src/bun.js/api/bun/ms.zig

```diff
@@ -0,0 +1,308 @@
+/// Parse a time string like "2d", "1.5h", "5m" to milliseconds
+pub fn parse(input: []const u8) ?f64 {
```

Comment:
Why return a float instead of an integer? Do we ever try to handle microseconds? There's going to be a loss of precision with floating points at `2^53`. That's about 285616y which I'm OK with being the upper threshold, but `u64` would've fit the bill just the same and arguably is slightly faster on the CPU since it doesn't go through the FPU